cmd/mr_create.go

```diff
@@ -193,6 +193,10 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 		}
 	}
 
+	if title == "" {
+		log.Fatal("aborting MR due to empty MR msg")
+	}
+
```

Comment:
This makes sense to me.

However:

```
		isWIP := strings.EqualFold(title[0:4], "wip:")
```

this probably still panics if title is not empty, but shorter than 4 characters?

Reply:
Should be fixed with the new patch.
Thanks Florian!